python/metrics/evaluation_tools/metrics/metrics.py

```diff
@@ -0,0 +1,247 @@
+"""
+==================
+Evaluation Metrics
+==================
+Convenience methods for computing common evaluation metrics.
+
+For a description of common evaluation metrics, see:
+
+http://www.eumetrain.org/data/4/451/english/courses/msgcrs/index.htm
+
+Functions
+---------
+compute_contingency_table
+probability_of_detection
+probability_of_false_detection
+probability_of_false_alarm
+threat_score
+frequency_bias
+percent_correct
+base_chance
+equitable_threat_score
+
+"""
+
+import pandas as pd
+from typing import Union
+
+def compute_contingency_table(
+    observed: pd.Series,
+    simulated: pd.Series
+    ) -> pd.Series:
+    """Compute components of a contingency table.
+        
+        Parameters
+        ----------
+        observed: pandas.Series, required
+            pandas.Series of boolean pandas.Categorical values indicating
+            observed occurences
+        simulated: pandas.Series, required
+            pandas.Series of boolean pandas.Categorical values indicating
+            simulated occurences
+            
+        Returns
+        -------
+        contingency_table: pandas.Series
+            pandas.Series of integer values keyed to pandas.Index([
+                'true_positive',
+                'false_positive,
+                'false_negative',
+                'true_negative'
+                ])
+        
+    """
+    # Cross tabulate
+    ctab = pd.crosstab(observed, simulated, dropna=False)
+
+    # Reformat
+    return pd.Series({
+        'true_positive' : ctab.loc[True, True],
+        'false_positive' : ctab.loc[True, False],
+        'false_negative' : ctab.loc[False, True],
+        'true_negative' : ctab.loc[False, False]
+        })
+
+def probability_of_detection(
```

Comment:
Wouldn't hurt to add `true_positive_key` and `false_negative_key` as keyword args with default values. I think that would help make the method intention more clear.